utxo.go

```diff
@@ -8,11 +8,12 @@ import (
 
 // UTXO an unspent transaction output, used for creating inputs
 type UTXO struct {
-	TxID           []byte
-	Vout           uint32
-	LockingScript  *bscript.Script
-	Satoshis       uint64
-	SequenceNumber uint32
+	TxID           []byte          `json:"txid"`
+	Vout           uint32          `json:"vout"`
+	LockingScript  *bscript.Script `json:"locking_script"`
+	Satoshis       uint64          `json:"satoshis"`
+	SequenceNumber uint32          `json:"sequence_number"`
+	Unlocker       *Unlocker
```

Comment:
@theflyingcodr am i missing anything or will it be fine that i add the `Unlocker` field here?